websites/R/RTL+/presence.ts

```diff
@@ -0,0 +1,154 @@
+const presence = new Presence({
+		clientId: "1033504954763198545",
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
```

Comment:
```suggestion
async function getStrings() {
	return presence.getStrings(
		{
		play: "general.playing",
		pause: "general.paused",
		}
	);
}

```

Reply:
Doesn't seem like this was resolved

Reply:
@EncryptedDev  https://github.com/PreMiD/Presences/blob/f4c5dea13b2af0aab930e3fbde123666ae9e333f/websites/R/RTL%2B/presence.ts#L4-L12